src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -154,6 +154,7 @@ export type Fiber = {
   // progressed work or if it is better to continue from the current state.
   progressedPriority: PriorityLevel,
   progressedWork: ProgressedWork,
+  newestWork: Fiber,
```

Comment:
Now we have another mostly synonymous term to add to the mix 🤡  current, in-progress, progressed, newest